### PR TITLE
Added health bar

### LINKED ISSUE #29: UI elements for player health

### PR DESCRIPTION
**Pull request by johnsonb2**
Fixes #29

### CODE REVIEW

File: public_html/src/MyGame/MyGame.js [hunk 1]
```diff
@@ -110,6 +118,7 @@ MyGame.prototype.draw = function () {
 
     this.mCamera.setupViewProjection();
     
+    this.mHobbesHealthBar.draw(this.mCamera);
```
Comment (Chris-Johnston) [inline public_html/src/MyGame/MyGame.js]: might make sense to draw this above all other objects

File: public_html/src/MyGame/MyGame.js [hunk 2]
```diff
@@ -65,9 +66,16 @@ MyGame.prototype.initialize = function () {
     this.mWorldBounds = new WorldBounds(centerPos, width, height);
     // Camera
     this.mCamera = new Camera(centerPos, width, [0, 0, 800, 600]);
-    this.mCamera.setBackgroundColor([0, 0, 0, 1]);
+    this.mCamera.setBackgroundColor([0, 0, 1, 1]);
```
Comment (Chris-Johnston) [inline public_html/src/MyGame/MyGame.js]: shouldn't matter, but not sure if relevant for this PR

File: public_html/src/MyGame/Objects/HealthBar.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    }
+}
+
+HealthBar.prototype.draw = function (camera) {
+    // Update positions of Renderables
+    var xform = this.mBackground.getXform();
+    var cameraPos = camera.getWCCenter();
+    var xPos = cameraPos[0] + this.mRelPos[0];
+    var yPos = cameraPos[1] + this.mRelPos[1];
+    xform.setPosition(xPos, yPos);
```
Comment (Chris-Johnston) [inline public_html/src/MyGame/Objects/HealthBar.js]: Seems like this should be moved into an update function

File: public_html/src/MyGame/MyGame.js (hunk 2 above)
Comment (johnsonb2) [reply public_html/src/MyGame/MyGame.js]: The background was black and I needed to see the black background of the health bar.

**Pull request merged**